file_handling.py

```diff
@@ -0,0 +1,60 @@
+import os.path
```

Comment:
I'd break this file to 2:
1. Create a storage object that internally stores and loads its content from a JSON file
2. Use the storage object to handle specific pokemon storage related functions